tests/test_annotations.py

```diff
@@ -0,0 +1,37 @@
+from ChildProject.projects import ChildProject, RecordingProfile
+from ChildProject.annotations import AnnotationManager
+from ChildProject.tables import IndexTable
+import pandas as pd
+import numpy as np
+import os
+
+def test_import():
+    project = ChildProject("examples/valid_raw_data")
+    project.import_data("output/annotations")
+    project = ChildProject("output/annotations")
+    am = AnnotationManager(project)
+
+    input_annotations = pd.read_csv('examples/valid_raw_data/raw_annotations/input.csv')
+
+    am.import_annotations(input_annotations)
+    am.read()
+    
+    assert am.annotations.shape[0] == input_annotations.shape[0], "imported annotations length does not match input"
+
+    assert all([
+        os.path.exists(os.path.join(project.path, 'annotations', f))
+        for f in am.annotations['annotation_filename'].tolist()
+    ]), "some annotations are missing"
+
+    errors, warnings = am.validate()
+    assert len(errors) > 0 or len(warnings) > 0, "malformed annotations detected"
+
```

Comment:
here, it would be good to test converted annotations agains the desired output in some way ! @alecristia

Reply:
are you asking me to list the tests? I'd be happy to - I know many ways in which these can be broken!

Reply:
Sure :)

Reply:
I do not think it is useful to differentiate them for now :)